test-app/src/components/debugger.tsx

```diff
@@ -23,6 +23,7 @@ import {
   tupleCV,
   someCV,
 } from '@stacks/transactions';
+import { TransactionsSelectors } from '../../../tests/integration/transactions.selectors';
```

Comment:
@ import

Reply:
It would work inside the test-app.

Reply:
can you try adding the paths? https://github.com/blockstack/stacks-wallet-web/blob/main/test-app/tsconfig.json#L27

Reply:
I can't get the path to be recognized. 🤔

Reply:
How have you been adding them?

Reply:
In the test-app tsconfig with the right route to the test folder (from the test-app src folder), right?

Reply:
This resolves it locally for me:
```diff
{
  "compilerOptions": {
    "target": "es2017",
    "module": "commonjs",
    "lib": ["es2017", "dom"],
    "jsx": "react",
    "sourceMap": true,
    "removeComments": true,
    "noEmit": false,
    "importHelpers": true,
    "downlevelIteration": true,
    "strict": true,
    "noImplicitAny": true,
    "strictNullChecks": true,
    "strictFunctionTypes": true,
    "strictBindCallApply": true,
    "strictPropertyInitialization": true,
    "noImplicitThis": true,
    "alwaysStrict": true,
    "skipLibCheck": true,
    "noUnusedLocals": true,
    "noUnusedParameters": true,
    "noImplicitReturns": true,
    "noFallthroughCasesInSwitch": true,
    "resolveJsonModule": true,
    "moduleResolution": "node",
    "paths": {
      "@store/*": ["./store/*"],
      "@store": ["./store/index"],
      "@dev/*": ["./dev/*"],
      "@components/*": ["./components/*"],
      "@containers/*": ["./containers/*"],
      "@common/*": ["./common/*"],
      "@cards/*": ["./components/cards/*"],
+     "@tests/*": ["../../tests/*"]
    },
    "baseUrl": "./src",
    "allowSyntheticDefaultImports": true,
    "esModuleInterop": true
  },
  "include": ["./src/**/*", "./tests/**/*", "./components/**/*", "./common/**/*"]
}
```

Reply:
and in the debugger: 
![Screen Shot 2021-07-07 at 11 10 48 AM](https://user-images.githubusercontent.com/11803153/124793846-03249080-df14-11eb-966d-64e067752da6.png)

Reply:
Really?! So weird, definitely tried that exactly. Let me try again, thx for checking it.

Reply:
![image](https://user-images.githubusercontent.com/6493321/124805144-be532680-df20-11eb-8095-44ffcdd84058.png)

Reply:
I shutdown VSCode and reopened ...looks like it fixed it. Very annoying! Haha